keyboards/windx65/config.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 thewerther (@thewerther)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
```

Comment:
This should all be moved to `info.json` file, as per the [PR checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs), and this file removed. You can use `qmk info -f json -kb windx65` to help with that.